tools/check-schemas

```diff
@@ -55,29 +55,37 @@ from zerver.lib import event_schema
 from zerver.lib.data_types import make_checker
 
 
-def get_event_checker(event: dict[str, Any]) -> Callable[[str, dict[str, Any]], None] | None:
+def get_event_checker(event: dict[str, Any]) -> Callable[[str, dict[str, Any]], None]:
+    # Follow the naming convention to find the event checker.
+    # Start by grabbing the event type.
     name = event["type"]
+
+    # Work around the fact that we send a "saved_snippets" (plural)
+    # event type for an event that really just sends a single value.
+    if name == "saved_snippets":
+        name = "saved_snippet"
```

Comment:
I'm not sure I quite understand why a workaround is required here, is there something to raise in https://chat.zulip.org/#narrow/channel/378-api-design/topic/saved.20replies/near/1942380 as a bug to fix? The original author is still quite active.

Reply:
It is not, as I pointed out [yesterday](https://github.com/zulip/zulip/pull/32865#discussion_r1900452685).

Reply:
Yup, it should definitely be easy to fix!!! Thanks for merging as is, but this is an ideal fix-forward cleanup.